# Translations template for nova.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the nova project.
#
# Translators:
# Xiao Xi LIU <liuxx@cn.ibm.com>, 2014
# 颜海峰 <yanheven@gmail.com>, 2014
# Andreas Jaeger <jaegerandi@gmail.com>, 2016. #zanata
msgid ""
msgstr ""
"Project-Id-Version: nova 14.0.0.0b3.dev750\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/openstack-i18n/\n"
"POT-Creation-Date: 2016-08-25 06:15+0000\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2016-03-08 01:29+0000\n"
"Last-Translator: Gaoxiao Zhu <zhu.gaoxiao@h3c.com>\n"
"Language: zh-CN\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 3.7.3\n"
"Language-Team: Chinese (China)\n"

msgid "\"Look for the VDIs failed"
msgstr "查找VDI失败"

#, python-format
msgid ""
"%(base)s virtual size %(base_size)s larger than flavor root disk size "
"%(size)s"
msgstr "%(base)s 虚拟大小 %(base_size)s 比云主机类型的根磁盘大小 %(size)s 大"

#, python-format
msgid ""
"%(scheme)s is registered as a module twice. %(module_name)s is not being "
"used."
msgstr "%(scheme)s作为模块被登陆两次。%(module_name)s 没有被使用。"

#, python-format
msgid ""
"Aggregate %(aggregate_id)s: unrecoverable state during operation on %(host)s"
msgstr "聚集 %(aggregate_id)s：在 %(host)s 上执行操作期间，处于不可恢复状态"

msgid "An error occurred while deallocating network."
msgstr "重新分配网络时发生错误。"

msgid "An error occurred while refreshing the network cache."
msgstr "刷新网络缓存时发生错误。"

#, python-format
msgid "Attaching PCI devices %(dev)s to %(dom)s failed."
msgstr "绑定PCI设备%(dev)s 到 %(dom)s 失败。"

msgid "Cannot cleanup migration files"
msgstr "无法清除迁移文件"

#, python-format
msgid "Cannot reboot instance: %s"
msgstr "无法重启实例：%s"

#, python-format
msgid "Caught error: %s"
msgstr "抓到错误：%s"

msgid "Compute driver option required, but not specified"
msgstr "需要计算驱动程序选项，但是未指定"

#, python-format
msgid "Connection to libvirt failed: %s"
msgstr "与 libvirt 的连接失败：%s"

#, python-format
msgid "Could not bind to %(host)s:%(port)s"
msgstr "无法绑定至 %(host)s:%(port)s"

msgid "Could not decode scheduler options"
msgstr "无法解码调度选项"

#, python-format
msgid ""
"Could not disconnect encrypted volume %(volume)s. If dm-crypt device is "
"still active it will have to be destroyed manually for cleanup to succeed."
msgstr ""
"不能断开加密卷%(volume)s。如果dm-crypt设备仍然处于活跃，为了成功清理，必须手"
"动销毁dm-crypt。"

#, python-format
msgid "Could not remove tmpdir: %s"
msgstr "未能除去临时目录：%s"

#, python-format
msgid "Could not start encryption for disk %(device)s: %(exception)s"
msgstr "对于磁盘 %(device)s 不能启动加密：%(exception)s"

#, python-format
msgid "Could not stat scheduler options file %(filename)s"
msgstr "无法获取调度选项文件 %(filename)s 大小"

msgid "Couldn't communicate with any cells"
msgstr "无法与任何单元通信"

#, python-format
msgid "Couldn't communicate with cell '%s'"
msgstr "无法与单元'%s’通信"

#, python-format
msgid "Couldn't lookup app: %s"
msgstr "无法查找应用：%s"

#, python-format
msgid "Couldn't unmount the GlusterFS share %s"
msgstr "不能卸载GlusterFS 共享 %s"

#, python-format
msgid "Couldn't unmount the NFS share %s"
msgstr "不能卸载 NFS 共享%s"

#, python-format
msgid "Couldn't unmount the Quobyte Volume at %s"
msgstr "不能卸载在%s的Quobyte卷"

#, python-format
msgid "Couldn't unmount the share %s"
msgstr "不能卸载共享%s"

#, python-format
msgid "Creating config drive failed with error: %s"
msgstr "创建配置驱动器失败，发生错误：%s"

msgid "DB error"
msgstr "数据库错误"

msgid "Destroy VM failed"
msgstr "销毁VM失败"

msgid "Destroy instance failed"
msgstr "销毁实例失败"

#, python-format
msgid "Detaching network adapter failed. Exception: %s"
msgstr "解绑网络适配器失败。异常：%s"

#, python-format
msgid ""
"Download handler '%(handler)s' raised an exception, falling back to default "
"handler '%(default_handler)s'"
msgstr ""
"下载处理 '%(handler)s'  引发一个异常，回滚到缺省处理 '%(default_handler)s'"

msgid "Download image error"
msgstr "下载镜像失败"

#, python-format
msgid "Driver failed to attach volume %(volume_id)s at %(mountpoint)s"
msgstr "驱动器未能在 %(mountpoint)s 上追加卷 %(volume_id)s"

msgid "Environment variable 'NETWORK_ID' must be set."
msgstr "必须设置环境变量'NETWORK_ID'。"

#, python-format
msgid "Error defining a domain with XML: %s"
msgstr "定义实体出错，XML：%s"

#, python-format
msgid "Error defining a secret with XML: %s"
msgstr "定义加密失败，XML：%s"

#, python-format
msgid "Error deleting conntrack entries for %s"
msgstr "删除跟踪连接项 %s"

#, python-format
msgid "Error deploying instance %(instance)s on baremetal node %(node)s."
msgstr "在 baremetal 节点 %(node)s 上部署实例 %(instance)s 时出错。"

#, python-format
msgid "Error enabling hairpin mode with XML: %s"
msgstr "启用发荚模式失败，XML：%s"

#, python-format
msgid "Error from last host: %(last_host)s (node %(last_node)s): %(exc)s"
msgstr "最后一个主机中出错：%(last_host)s（节点 %(last_node)s）：%(exc)s"

#, python-format
msgid "Error from libvirt during destroy. Code=%(errcode)s Error=%(e)s"
msgstr "销毁时发生错误。Code=%(errcode)s Error=%(e)s"

#, python-format
msgid "Error from libvirt during undefine. Code=%(errcode)s Error=%(e)s"
msgstr "在 undefine 时 libvirt 发生错误。Code=%(errcode)s Error=%(e)s"

#, python-format
msgid "Error from libvirt during unfilter. Code=%(errcode)s Error=%(e)s"
msgstr "在 unfilter 期间，libvirt 中出错。Code=%(errcode)s Error=%(e)s"

#, python-format
msgid "Error injecting data into image %(img_id)s (%(e)s)"
msgstr "将数据插入到映像 %(img_id)s (%(e)s) 中时出错"

#, python-format
msgid "Error instantiating %(provider)s: %(exception)s"
msgstr "实例化%(provider)s出错：%(exception)s"

#, python-format
msgid "Error launching a defined domain with XML: %s"
msgstr "启动定义的实体失败，XML：%s"

msgid "Error locating next hop for message"
msgstr "定位消息的下一跳发生错误"

msgid "Error locating next hops for message"
msgstr "定位消息的下一跳发生错误"

msgid ""
"Error occurred during volume_snapshot_create, sending error status to Cinder."
msgstr "在卷快照创建期间发生错误，发送错误状态到Cinder。"

msgid ""
"Error occurred during volume_snapshot_delete, sending error status to Cinder."
msgstr "在volume_snapshot_delete期间发生错误，发送错误状态到Cinder。"

#, python-format
msgid "Error on '%(path)s' while checking direct I/O: '%(ex)s'"
msgstr "检查直接 I/O 时，在“%(path)s”上出错：“%(ex)s”"

#, python-format
msgid ""
"Error preparing deploy for instance %(instance)s on baremetal node %(node)s."
msgstr "在裸节点%(node)s 为实例%(instance)s准备部署失败。"

msgid "Error processing message locally"
msgstr "本地消息处理发生错误"

#, python-format
msgid "Error releasing DHCP for IP %(address)s with MAC %(mac_address)s"
msgstr "释放带有MAC %(mac_address)s 的IP %(address)s 的DHCP时出错"

#, python-format
msgid "Error scheduling instances %(instance_uuids)s"
msgstr "调度实例 %(instance_uuids)s 时出错"

msgid "Error sending message to next hops."
msgstr "发送消息到下一跳失败"

#, python-format
msgid "Error starting xvp: %s"
msgstr "启动xvp发生错误：%s"

msgid "Error trying to Rescue Instance"
msgstr "尝试救援实例时发生错误"

msgid "Error trying to reschedule"
msgstr "尝试重新调度时出错"

#, python-format
msgid "Error updating resources for node %(node)s."
msgstr "更新节点 %(node)s资源时发生错误。"

msgid "Error waiting for responses from neighbor cells"
msgstr "等待来自相邻cell的响应发生错误"

#, python-format
msgid "Error while trying to clean up image %s"
msgstr "尝试清除镜像%s时出错"

#, python-format
msgid "Error writing to %(path)s: %(exception)s"
msgstr "写入 %(path)s: %(exception)s 出错"

#, python-format
msgid "Error: %s"
msgstr "错误：%s"

#, python-format
msgid "Exception dispatching event %(event)s: %(ex)s"
msgstr "分派事件 %(event)s 时发生异常：%(ex)s"

#, python-format
msgid "Exception handling resource: %s"
msgstr "处理资源时发生异常：%s"

msgid "Exception in string format operation"
msgstr "字符串格式操作中发生异常"

msgid "Exception loading extension"
msgstr "加载扩展异常"

#, python-format
msgid "Exception running %(name)s %(type)s-hook: %(obj)s"
msgstr "运行%(name)s %(type)s-hook: %(obj)s时发生异常"

msgid "Exception while creating table."
msgstr "创建表时发生异常。"

#, python-format
msgid "Exception while waiting completion of volume snapshots: %s"
msgstr "等待卷快照：%s 完成时发生异常"

#, python-format
msgid "Extending virtual disk failed with error: %s"
msgstr "扩展虚拟磁盘失败，错误：%s"

#, python-format
msgid "Failed removing bridge device: '%s'"
msgstr "未能移除桥接设备：%s"

#, python-format
msgid "Failed removing net device: '%s'"
msgstr "未能移除网络设备：%s "

msgid "Failed storing info cache"
msgstr "存储信息高速缓存失败"

msgid ""
"Failed to allocate PCI devices for instance. Unassigning devices back to "
"pools. This should not happen, since the scheduler should have accurate "
"information, and allocation during claims is controlled via a hold on the "
"compute node semaphore"
msgstr ""
"无法给实例分配PCI设备。取消分配设备回池。这本不该发生，因为调度器应该有精准的"
"信息和通过对计算节点信号的保持而控制要求资源过程中的分配。"

msgid "Failed to allocate network(s)"
msgstr "无法分配网络(s)"

#, python-format
msgid "Failed to attach %(volume_id)s at %(mountpoint)s"
msgstr "未能在 %(mountpoint)s 上追加 %(volume_id)s"

#, python-format
msgid "Failed to attach volume at mountpoint: %s"
msgstr "在挂载点%s绑定卷失败"

#, python-format
msgid "Failed to change vm state of %(instance_name)s to %(req_state)s"
msgstr "未能将虚拟机 %(instance_name)s 状态更改为%(req_state)s"

msgid "Failed to check if instance shared"
msgstr "无法查询实例是否共享"

#, python-format
msgid "Failed to cleanup directory %(target)s: %(e)s"
msgstr "未能清除目录 %(target)s：%(e)s"

msgid "Failed to cleanup snapshot."
msgstr "无法清除快照"

#, python-format
msgid "Failed to commit reservations %s"
msgstr "无法交付预留 %s"

msgid "Failed to complete a deletion"
msgstr "无法完成删除操作"

#, python-format
msgid "Failed to connect to volume %(volume_id)s with volume at %(mountpoint)s"
msgstr "未能将 %(mountpoint)s 的卷连接到卷 %(volume_id)s"

#, python-format
msgid ""
"Failed to copy cached image %(source)s to %(dest)s for resize: %(error)s"
msgstr "为调整大小，从%(source)s 到 %(dest)s复制缓存镜像失败：%(error)s"

msgid "Failed to deallocate networks"
msgstr "释放网络失败"

#, python-format
msgid "Failed to decrypt %(image_location)s to %(image_path)s"
msgstr "解密 %(image_location)s 到 %(image_path)s失败"

#, python-format
msgid "Failed to destroy instance: %s"
msgstr "未能销毁实例：%s"

#, python-format
msgid "Failed to detach %(device_name)s. Exception: %(exc)s"
msgstr "解绑%(device_name)s失败。异常：%(exc)s"

#, python-format
msgid "Failed to detach volume %(volume_id)s from %(mp)s"
msgstr "未能从 %(mp)s 断开与卷 %(volume_id)s 的连接"

#, python-format
msgid "Failed to download %(image_location)s to %(image_path)s"
msgstr "把 %(image_location)s 下载到 %(image_path)s失败"

msgid "Failed to fetch glance image"
msgstr "未能访存 Glance 映像"

#, python-format
msgid "Failed to find an SR for volume %s"
msgstr "为卷%s查找SR失败"

#, python-format
msgid "Failed to forget the SR for volume %s"
msgstr "为卷%s 断开SR失败"

#, python-format
msgid "Failed to generate usage audit for instance on host %s"
msgstr "未能为主机 %s 上的实例生成使用量审计"

#, python-format
msgid "Failed to get compute_info for %s"
msgstr "为 %s 获取compute_info失败"

msgid "Failed to get default networks"
msgstr "获取默认网络失败"

#, python-format
msgid ""
"Failed to get instance id for metadata request, provider %(provider)s "
"networks %(networks)s requester %(requester)s. Error: %(error)s"
msgstr ""
"从元数据请求中获取实例id失败，提供方 %(provider)s 网络 %(networks)s 请求方 "
"%(requester)s。错误：%(error)s"

#, python-format
msgid "Failed to get metadata for instance id: %s"
msgstr "未能针对实例标识获取元数据：%s"

msgid "Failed to get nw_info"
msgstr "未能获取 nw_info"

#, python-format
msgid "Failed to instantiate the download handler for %(scheme)s"
msgstr "未能实例化地下载处理程序 %(scheme)s"

#, python-format
msgid ""
"Failed to kill process %(pid)s due to %(reason)s, while deleting the "
"instance."
msgstr "删除实例时，杀死进程 %(pid)s 失败，原因是 %(reason)s 。"

#, python-format
msgid "Failed to load %(cfgfile)s: %(ex)s"
msgstr "加载%(cfgfile)s: %(ex)s失败"

#, python-format
msgid "Failed to load %s"
msgstr "加载 %s 失败"

#, python-format
msgid "Failed to load %s-api"
msgstr "加载 %s-api 失败"

#, python-format
msgid ""
"Failed to mount container filesystem '%(image)s' on '%(target)s': %(errors)s"
msgstr "未能在“%(target)s”上安装容器文件系统“%(image)s”：%(errors)s"

msgid "Failed to notify cells of BDM destroy."
msgstr "通知单元有关BDM销毁的事项失败"

msgid "Failed to notify cells of BDM update/create."
msgstr "通知单元有关BDM更新/创建事项失败"

msgid "Failed to notify cells of bw_usage update"
msgstr "未能通知单元有关 bw_usage 更新的事项"

msgid "Failed to notify cells of instance fault"
msgstr "未能通知单元有关实例故障的事项"

msgid "Failed to notify cells of instance info cache update"
msgstr "未能通知单元有关实例信息高速缓存更新的事项"

#, python-format
msgid "Failed to remove %(base_file)s, error was %(error)s"
msgstr "删除 %(base_file)s 失败，错误是 %(error)s"

#, python-format
msgid "Failed to remove snapshot for VM %s"
msgstr "移除 VM %s 快照失败"

#, python-format
msgid "Failed to request Ironic to provision instance %(inst)s: %(reason)s"
msgstr "请求Ironic准备实例%(inst)s失败：%(reason)s"

#, python-format
msgid ""
"Failed to retrieve encryption metadata for volume %(volume_id)s: "
"%(exception)s"
msgstr "为卷%(volume_id)s 获取加密元数据失败：%(exception)s"

msgid "Failed to retrieve ephemeral encryption key"
msgstr "获取瞬时加密关键字失败"

msgid "Failed to revert crashed migration"
msgstr "未能还原已崩溃的迁移"

#, python-format
msgid "Failed to roll back reservations %s"
msgstr "无法回滚预留 %s"

msgid "Failed to rollback quota for failed finish_resize"
msgstr "未能对失败的finish_resize进行回滚配额"

#, python-format
msgid "Failed to send message to cell: %(next_hop)s"
msgstr "发送消息到单元：%(next_hop)s失败"

msgid "Failed to send state update notification"
msgstr "未能发送状态更新通知"

msgid "Failed to send updated snapshot status to volume service."
msgstr "发送更新快照状态到卷服务失败"

msgid "Failed to set instance name using multi_instance_display_name_template."
msgstr "未能使用 multi_instance_display_name_template 来设置实例名称。"

#, python-format
msgid "Failed to start %(name)s on %(host)s:%(port)s with SSL support"
msgstr "未能通过 SSL 支持对 %(host)s:%(port)s 启动 %(name)s"

msgid "Failed to start instance"
msgstr "无法启动实例"

msgid "Failed to stop instance"
msgstr "无法停止实例"

#, python-format
msgid "Failed to swap volume %(old_volume_id)s for %(new_volume_id)s"
msgstr "未能将卷 %(old_volume_id)s 替换为 %(new_volume_id)s"

msgid "Failed to teardown container filesystem"
msgstr "未能卸载容器文件系统"

msgid "Failed to umount container filesystem"
msgstr "未能取消挂载容器文件系统"

msgid "Failed to unpause instance"
msgstr "无法取消暂停实例"

#, python-format
msgid "Failed to untar %(image_location)s to %(image_path)s"
msgstr "解包 %(image_location)s 到 %(image_path)s 失败"

msgid "Failed to update usages bulk deallocating floating IP"
msgstr "未能更新使用量释放浮动IP"

msgid "Failed to update usages deallocating fixed IP"
msgstr "未能更新使用量释放固定 IP"

msgid "Failed to update usages deallocating floating IP"
msgstr "未能更新使用量释放浮动 IP"

msgid "Failed to update usages deallocating network."
msgstr "未能更新使用量释放网络。"

msgid "Failed to update usages deallocating security group"
msgstr "未能更新使用量释放安全组"

msgid "Failed to update usages deallocating server group"
msgstr "未能更新使用量释放服务器组"

#, python-format
msgid "Failed to upload %(image_location)s to %(image_path)s"
msgstr "上传 %(image_location)s 到 %(image_path)s 失败"

msgid "Failed to write configuration file"
msgstr "未能写入配置文件"

msgid "Failed to write inbound.csr"
msgstr "未能写入 inbound.csr"

msgid "Failed while plugging ib hostdev vif"
msgstr "插入ib 主机设备 vif 失败"

msgid "Failed while plugging vif"
msgstr "插入vif时失败"

msgid "Failed while unplugging ib hostdev vif"
msgstr "拔出 ib 主机 vif 失败"

msgid "Failed while unplugging vif"
msgstr "拔出 vif 时发生故障"

msgid "Failure prepping block device"
msgstr "准备块设备失败"

msgid "Failure while cleaning up attached VDIs"
msgstr "清除已连接的 VDI 时，发生故障"

#, python-format
msgid "Fatal Exception running %(name)s %(type)s-hook: %(obj)s"
msgstr "运行%(name)s %(type)s-hook: %(obj)s时发生致命错误"

msgid "Guest does not have a console available"
msgstr "Guest 没有控制台可用"

msgid "Host is member of a pool, but DB says otherwise"
msgstr "主机是池的成员，但数据库则不然"

#, python-format
msgid ""
"Hostname has changed from %(old)s to %(new)s. A restart is required to take "
"effect."
msgstr "主机名从%(old)s 变为 %(new)s。必须重启才能生效。"

#, python-format
msgid "Image size %(size)d exceeded flavor allowed size %(allowed_size)d"
msgstr "镜像大小 %(size)d 超出云主机类型允许的大小 %(allowed_size)d"

#, python-format
msgid ""
"Instance compute service state on %s expected to be down, but it was up."
msgstr "实例计算服务状态 %s 预计为down，但是却是up。"

msgid "Instance failed block device setup"
msgstr "实例块设备设置失败"

#, python-format
msgid "Instance failed network setup after %(attempts)d attempt(s)"
msgstr "在%(attempts)d次尝试后,实例网络设置还是失败(嘀)"

msgid "Instance failed to spawn"
msgstr "实例生产失败"

#, python-format
msgid "Interface %s not found"
msgstr "没有找到接口%s"

#, python-format
msgid "Invalid server_string: %s"
msgstr "server_string 无效：%s"

#, python-format
msgid "Live Migration failure: %s"
msgstr "在线迁移失败：%s"

msgid "Live migration failed."
msgstr "在线迁移失败。"

msgid "Migrate Receive failed"
msgstr "Migrate Receive 失败"

msgid "Migrate Send failed"
msgstr "Migrate Send 失败"

#, python-format
msgid "Migration %s is not found during confirmation"
msgstr "确认时未发现迁移%s"

#, python-format
msgid ""
"Migration of instance %(instance_id)s to host %(dest)s unexpectedly failed."
msgstr "从实例 %(instance_id)s 迁移到主机 host %(dest)s 意外失败。"

msgid "Migration operation has aborted"
msgstr "放弃迁移操作"

#, python-format
msgid "NBD error - previous umount did not cleanup /var/lock/qemu-nbd-%s."
msgstr "NBD错误 - 之前取消挂载没有清理 /var/lock/qemu-nbd-%s。"

#, python-format
msgid ""
"NOT IMPLEMENTED: The call to %(method)s is not supported by the agent. args="
"%(args)r"
msgstr "未实现：对 %(method)s 的调用不受代理支持。args=%(args)r"

msgid "Network driver option required, but not specified"
msgstr "需要网络驱动程序选项，但是未指定"

#, python-format
msgid "Neutron Error adding rules to security group %s"
msgstr "添加规则到安全组 %s 时发生Neutron错误"

#, python-format
msgid "Neutron Error creating security group %s"
msgstr "创建安全组 %s 时发生Neutron错误"

#, python-format
msgid "Neutron Error getting security group %s"
msgstr "获取安全组 %s 时发生Neutron错误"

msgid "Neutron Error getting security groups"
msgstr "获取安全组时发生Neutron错误"

#, python-format
msgid "Neutron Error unable to delete %s"
msgstr "无法删除 %s 的Neutron错误"

#, python-format
msgid "Neutron Error updating security group %s"
msgstr "更新安全组 %s 时发生Neutron错误"

msgid "Neutron Error:"
msgstr "Neutron错误"

#, python-format
msgid "Neutron Error: %s"
msgstr "Neutron错误：%s"

#, python-format
msgid "Neutron Reported failure on event %(event)s for instance %(uuid)s"
msgstr "对于实例%(uuid)s的事件%(event)s，Neurton报告失败"

#, python-format
msgid "Neutron error creating port on network %s"
msgstr "在网络 %s 上创建端口发生Neutron错误"

#, python-format
msgid "No compute node record for host %s"
msgstr "没有host %s 的计算节点记录"

#, python-format
msgid "No save handler for %s"
msgstr "未保存对 %s 的处理"

msgid "Not all properties needed are implemented in the compute driver"
msgstr "在计算驱动程序中未实现所有必须的属性"

msgid ""
"Periodic sync_power_state task had an error while processing an instance."
msgstr "处理实例时定期任务sync_power_state发生错误"

msgid "Periodic task failed to offload instance."
msgstr "定期任务未能卸载实例。"

msgid "Plugin config_file get_val failed"
msgstr "插入config_file get_val失败"

#, python-format
msgid "Pool-Join failed: %s"
msgstr "Pool-Join 失败：%s"

#, python-format
msgid "Pool-eject failed: %s"
msgstr "Pool-eject 失败：%s"

#, python-format
msgid "Pool-set_name_label failed: %s"
msgstr "Pool-set_name_label 失败：%s"

#, python-format
msgid "Pre live migration failed at %s"
msgstr "预热迁移失败 %s"

msgid "Read/Write data failed"
msgstr "读/写数据失败"

#, python-format
msgid "Requested VM Generation %s, but provided VHD instead of VHDX."
msgstr "请求VM Generation %s，但是返回VHD而不是VHDX。"

#, python-format
msgid "Returning exception %s to caller"
msgstr "正在将异常 %s 返回至调用者"

msgid "Service error occurred during cleanup_host"
msgstr "在cleanup_host中发生服务错误"

msgid "Setting instance vm_state to ERROR"
msgstr "设置实例的vm_state为ERROR"

msgid "Shutting down VM (cleanly) failed."
msgstr "关闭虚拟机(软)失败。"

msgid "Shutting down VM (hard) failed"
msgstr "关闭虚拟机(硬)失败"

msgid ""
"Something wrong happened when trying to delete snapshot from shelved "
"instance."
msgstr "在试图从废弃的实例中删除快照时发生了错误。"

#, python-format
msgid "TIMEOUT: The call to %(method)s timed out. args=%(args)r"
msgstr "超时：对 %(method)s 的调用超时。args=%(args)r"

#, python-format
msgid "Temporary directory is invalid: %s"
msgstr "临时目录无效：%s"

#, python-format
msgid "The Quobyte volume at %s is still in use."
msgstr "在%s 的Quobyte 卷仍然在用。"

#, python-format
msgid ""
"The agent call to %(method)s returned an invalid response: %(ret)r. args="
"%(args)r"
msgstr "代理调用%(method)s，返回一个无效响应：%(ret)r. args=%(args)r"

#, python-format
msgid "The call to %(method)s returned an error: %(e)s."
msgstr "对 %(method)s 的调用返回错误：%(e)s。"

#, python-format
msgid "The call to %(method)s returned an error: %(e)s. args=%(args)r"
msgstr "对 %(method)s 的调用返回了错误：%(e)s。args=%(args)r"

msgid "Transfer data failed"
msgstr "传输数据失败"

#, python-format
msgid "Unable to access floating IP %s"
msgstr "无法访问浮动IP %s"

#, python-format
msgid "Unable to access floating IP for %s"
msgstr "无法访问 %s的浮动IP"

msgid "Unable to access the rescue disk"
msgstr "不能访问营救盘"

#, python-format
msgid "Unable to attach volume to instance %s"
msgstr "无法附加卷到实例 %s"

#, python-format
msgid "Unable to clear device ID for port '%s'"
msgstr "无法清除端口'%s'的设备ID"

msgid "Unable to create VM snapshot, failing volume_snapshot operation."
msgstr "无法创建VM快照，卷快照操作失败。"

msgid ""
"Unable to create quiesced VM snapshot, attempting again with quiescing "
"disabled."
msgstr "无法创建停顿VM的快照，禁用停顿后再尝试一次。"

msgid "Unable to destroy VBD"
msgstr "不能销毁VBD"

#, python-format
msgid "Unable to execute %(cmd)s. Exception: %(exception)s"
msgstr "无法执行 %(cmd)s。发生异常：%(exception)s"

msgid "Unable to find SR from VBD"
msgstr "不能从VBD找到SR"

msgid "Unable to find SR from VDI"
msgstr "不能从VDI找到SR"

msgid "Unable to get record of VDI"
msgstr "不能获取VDI记录"

msgid "Unable to get updated status"
msgstr "无法获取已更新的状态"

msgid "Unable to introduce VDI for SR"
msgstr "不能为SR引入VDI"

msgid "Unable to introduce VDI on SR"
msgstr "不能在SR上引入VDI"

msgid "Unable to load the virtualization driver"
msgstr "无法加载虚拟驱动"

#, python-format
msgid "Unable to migrate VM %(vm_ref)s from %(host)s"
msgstr "不能从%(host)s 迁移虚拟机%(vm_ref)s"

#, python-format
msgid ""
"Unable to obtain RRD XML for VM %(vm_uuid)s with server details: %(server)s."
msgstr "无法为含服务器详细信息的虚拟机 %(vm_uuid)s 获取RRD XML：%(server)s。"

#, python-format
msgid "Unable to parse rrd of %s"
msgstr "不能解析 %s 的rrd"

#, python-format
msgid "Unable to retrieve storage policy with name %s"
msgstr "不能获取名称 %s 的存储策略"

#, python-format
msgid "Unable to set up pool: %s."
msgstr "不能建立池：%s"

msgid "Unable to unplug VBD"
msgstr "不能拔出VBD"

#, python-format
msgid "Unable to update instance VNIC index for port %s."
msgstr "无法更新实例VNIC的索引的端口 %s"

msgid "Unexpected build failure, not rescheduling build."
msgstr "意外构筑失败，未重新安排构筑。"

#, python-format
msgid "Unexpected error while checking process %(pid)s."
msgstr "检查进程 %(pid)s 时，发生未知错误。"

msgid "Unexpected error while reporting service status"
msgstr "当报告服务状态时发生未知错误"

msgid "Unexpected exception in API method"
msgstr "在API方法中发生未预料的异常"

#, python-format
msgid "Unknown cell '%(cell_name)s' when trying to update capabilities"
msgstr "尝试更新功能时，遇到未知单元“%(cell_name)s”"

#, python-format
msgid "Unknown cell '%(cell_name)s' when trying to update capacities"
msgstr "尝试更新容量时，遇到未知单元“%(cell_name)s”"

#, python-format
msgid "Unknown content in connection_info/access_mode: %s"
msgstr "在connection_info/access_mode: %s 中有未知内容"

msgid "Unshelve attempted but an error has occurred"
msgstr "试图取消废弃但是发生了错误"

msgid "Unshelve attempted but vm_state not SHELVED or SHELVED_OFFLOADED"
msgstr "试图取消废弃但是vm_state不是SHELVED或者SHELVED_OFFLOADED"

msgid "Vifs plug failed"
msgstr "Vifs插入失败"

#, python-format
msgid "Volume %s is associated with the instance but no SR was found for it"
msgstr "卷%s 与实例关联，但是没有为它找到SR"

#, python-format
msgid "Volume is not writable. Please broaden the file permissions. Mount: %s"
msgstr "卷不是可写。请扩大文件权限。挂载： %s"

#, python-format
msgid ""
"When loading the module %(module_str)s the following error occurred: %(ex)s"
msgstr "在加载 %(module_str)s 模块时发生以下错误：%(ex)s"

msgid ""
"XenAPI is unable to find a Storage Repository to install guest instances on. "
"Please check your configuration (e.g. set a default SR for the pool) and/or "
"configure the flag 'sr_matching_filter'."
msgstr ""
"XenAPI 不能找到存储仓库来安装客户机实例。请检查配置(比如设置池的缺省SR)，且/"
"或配置标记'sr_matching_filter'。"

msgid "_migrate_disk_resizing_down failed. Restoring orig vm"
msgstr "_migrate_disk_resizing_down失败。恢复原始VM"

#, python-format
msgid "_migrate_disk_resizing_up failed. Restoring orig vm due_to: %s."
msgstr "_migrate_disk_resizing_up failed失败。恢复原始VM，原因是：%s。"

#, python-format
msgid "allocate_port_for_instance returned %(ports)s ports"
msgstr "allocate_port_for_instance 返回端口 %(ports)s"

msgid "attaching network adapter failed."
msgstr "连接网络适配器失败。"

msgid "detaching network adapter failed."
msgstr "未能与网络适配器断开连接。"

msgid "error during stop() in sync_power_state."
msgstr "在 stop()（处于 sync_power_state）期间出错。"

#, python-format
msgid "error opening rbd image %s"
msgstr "打开rbd镜像%s 出错"

#, python-format
msgid "image %(volume)s in pool %(pool)s has snapshots, failed to remove"
msgstr "池%(pool)s 中的镜像 %(volume)s 有快照，移除失败"

#, python-format
msgid "kill -HUP dnsmasq threw %s"
msgstr "终止-HUP dnsmasq 进程时抛出 %s"

#, python-format
msgid "killing radvd threw %s"
msgstr "杀掉进程 radvd 时抛出 %s"

msgid "nbd module not loaded"
msgstr "未装入 nbd 模块"

msgid "set_admin_password failed"
msgstr "set_admin_password失败"

#, python-format
msgid ""
"status is UNKNOWN from vm_state=%(vm_state)s task_state=%(task_state)s. Bad "
"upgrade or db corrupted?"
msgstr ""
"根据 vm_state=%(vm_state)s task_state=%(task_state)s，状态为未知。是升级不正"
"确，还是数据库已损坏？"
